### PR TITLE
Hologram Touchup (Init savings edition)

### PR DESCRIPTION
**Pull request by LemonInTheDark**
### Polishes and Reworks Holograms

Hologram generation currently involves a bunch of icon operations, which are slow. 
Not to mention a series of get flats for the human models, which is even worse.

We lose 0.05 seconds of init to em off just the 2 RCD holograms. it hurts man.

So instead, let's use filters and render steps to achive the same effect.

While I'm here I'll dim the holo light and make it blue, make the hologram and its beam emissive (so they glow), and do some fenangling with move_hologram() (it doesn't clear the hologram off failure anymore, instead relying on callers to do that) to ensure holocalls can't be accidentially ended by moving out of the area.

Ah and I added RESET_ALPHA to the emissive appearance flags, cause the alpha does override and fuck with color rendering, which ends up looking dumb. If we're gonna support this stuff it should be first class not accidential.

### Makes Static Not Shit

While I'm here (since holograms see static) lets ensure the static plane is always visible if you're seeing through an ai eye.

The old solution was limited to applying it to JUST ais, which isn't satisfactory for this sort of thing and missed a LOT of cases (I didn't really get how ai eyes worked before I'ma be honest)

I'm adding a signal off the hud for it detecting a change in its eye here. 
This is semi redundant, but avoids unneeded dupe work, so I'm ok with it.

The pipeline here is less sane then I'd like, but it works and that's enough

## Why It's Good For The Game

dreamseeker_zMiLXzlZ2X
More pretty, better ux, **static works**

## Changelog
:cl:
add: Holograms glow now, pokes at the lighting for holocalls in general a bit to make em nicer.
qol: You can no longer accidentally end a holocall (as a non ai) by leaving the area. Felt like garbage
fix: Fixes static rendering improperly if viewed by a non ai
/:cl:

### CODE REVIEW

File: code/__DEFINES/dcs/signals/signals_hud.dm [hunk 1]
```diff
@@ -1,2 +1,5 @@
+/// Sent from /datum/hud/proc/on_eye_change(): (atom/old_eye, atom/new_eye)
+#define COSMIG_HUD_EYE_CHANGED "hud_eye_changed"
```
Comment (san7890) [inline code/__DEFINES/dcs/signals/signals_hud.dm:2]:
```suggestion
#define COMSIG_HUD_EYE_CHANGED "hud_eye_changed"
```

will probably also want to find and replace this everywhere

File: code/_onclick/hud/rendering/plane_master.dm [hunk 2]
```diff
@@ -550,18 +550,31 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
...
 /atom/movable/screen/plane_master/camera_static/show_to(mob/mymob)
-	// If we aren't an AI, we have no need for this plane master (most of the time, ai eyes are weird and annoying)
-	if(force_hidden && isAI(mymob))
-		unhide_plane(mymob)
 	. = ..()
 	if(!.)
 		return
-	if(isAI(mymob))
+	var/datum/hud/our_hud = home.our_hud
+	if(!our_hud)
```
Comment (san7890) [inline code/_onclick/hud/rendering/plane_master.dm:559]: could we use isnull here instead for the swag explicit readability

**Review by san7890 — CHANGES_REQUESTED** [review]

File: code/__DEFINES/dcs/signals/signals_hud.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/__DEFINES/dcs/signals/signals_hud.dm:2]: I uh, oh I fucked up the S and M, thanks

File: code/_onclick/hud/rendering/plane_master.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/_onclick/hud/rendering/plane_master.dm:559]: fine, for you anything

**Pull request merged**